pyaedt/edb_core/EDB_Data.py

```diff
@@ -2663,9 +2663,32 @@ def refdes(self):
         """
         return self.edbcomponent.GetName()
 
+    @property
+    def is_enabled(self):
+        """Check if the current object is enabled.
+
+        Returns
+        -------
+        bool
+            ``True`` if current object is enabled, ``False`` otherwise.
+        """
+        return self.edbcomponent.GetComponentProperty().IsEnabled()
```

Comment:
@ring630 .
This is causing the IronPython tests failure.
IronPython is less permissive that Python with some incorrect types provided.
![image](https://user-images.githubusercontent.com/87315832/178173735-74c4f57d-af58-41cd-9e35-91ede3b5a39f.png)

You can debug in Python on your local machine if it is more convenient for you and check the type returned by `self.edbcomponent.GetComponentProperty()`.